src/components/MessageCard/utils/MessageCard.hooks.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+const TRANSITION_DURATION = 200
```

Comment:
Should we maybe make it customizable via optional property to the hook? It can take this value as default. We might not  need it now, but I can see this might potentially be useful in other uses (and maybe it makes sense to move it to some utils folder, as it's not very Messages specific). What do you think?